File: Iniciante/URI 1012.py
retangulo = None
quadrado = None
trapezio = None
circulo = None
triangulo = None
b = None
a = None
c = None
lista = None

def read_line():
  try:
    # read for Python 2.x
    return raw_input()
  except NameError:
    # read for Python 3.x
    return input()

lista = read_line().split(" ")
a = float((lista[0]))
b = float((lista[1]))
c = float((lista[2]))
triangulo = a * c / 2.0
circulo = 3.14159 * (c * c)
trapezio = ((a + b) * c) / 2.0
quadrado = b * b
retangulo = a * b
print(str("TRIANGULO: ") + str("{:0.3f}".format(triangulo)))
print(str("CIRCULO: ") + str("{:0.3f}".format(circulo)))
print(str("TRAPEZIO: ") + str("{:0.3f}".format(trapezio)))
print(str("QUADRADO: ") + str("{:0.3f}".format(quadrado)))
print(str("RETANGULO: ") + str("{:0.3f}".format(retangulo)))

File: Iniciante/URI 1012.lua
line = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line[n] = i
end
a = tonumber(line[1])
b = tonumber(line[2])
c = tonumber(line[3])

triangulo = a * c / 2.0
circulo = 3.14159 * (c * c)
trapezio = ((a  + b) * c  ) / 2.0
quadrado = b * b
retangulo = a * b

print(string.format("TRIANGULO: %.3f", triangulo))
print(string.format("CIRCULO: %.3f", circulo))
print(string.format("TRAPEZIO: %.3f", trapezio))
print(string.format("QUADRADO: %.3f", quadrado))
print(string.format("RETANGULO: %.3f", retangulo)) 